core/src/events.rs

```diff
@@ -33,7 +33,7 @@ pub enum CollisionEvent {
 }
 
 /// Collision data concerning one of the two entity that collided
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Reflect)]
```

Comment:
Why add these two traits?

Reply:
I believe the `PartialEq` is only there for the tests (so not really needed), but `Collisions` is `Reflect`, so `CollisionData` needs to be `Reflect` as well (unless we make a custom implementation).